### PR TITLE
Fix ingress layout

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: helm_deploy/prisoner-content-hub-frontend/templates/ingress.yaml [hunk 1]
```diff
@@ -14,11 +14,11 @@ metadata:
 spec:
   tls:
   {{- range .Values.ingress.hosts }}
+  {{- $certSecret := .cert_secret -}}
```
Comment (psoleckimoj) [inline helm_deploy/prisoner-content-hub-frontend/templates/ingress.yaml:17]: I understand more formatting now and I always wanted these here 😁

Comment (spikeheap) [reply helm_deploy/prisoner-content-hub-frontend/templates/ingress.yaml:17]: Ahhh yeah, I've been bitten by that a couple of times 🤦. Elegant helm indentation != valid YAML ☹️.

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Review by spikeheap — APPROVED** [review]
Looks good, and I've felt that pain! 

I _think_ the indentation on L21 was correct (it's 2 deeper than the previous item), but YAML copes with the extra level as long as it's consistent so whichever feels nicer is good ✅

**Pull request merged**